command/debug.go

```diff
@@ -773,7 +773,11 @@ func (c *DebugCommand) collectPprof(ctx context.Context) {
 			wg.Add(1)
 			go func(target string) {
 				defer wg.Done()
-				data, err := pprofTarget(ctx, c.cachedClient, target, nil)
+
+				seconds := int(c.flagInterval.Seconds())
+				secStr := strconv.Itoa(seconds)
+
+				data, err := pprofTarget(ctx, c.cachedClient, target, url.Values{"seconds": []string{secStr}})
```

Comment:
Does this `url.Values{}` thing get used anywhere?

Reply:
It's used in the pprof handler: https://cs.opensource.google/go/go/+/refs/tags/go1.22.4:src/net/http/pprof/pprof.go;l=252

However, there's some difference between the way this behaves and our tests expect the named profile targets to behave, so I'll need to look into that a bit more before we can merge.